autogen/defaultModel.json

```diff
@@ -0,0 +1,27 @@
+{
+    "system": {
+        "target": "audiomini",
+        "sampleClockFrequency": 48000,
+        "systemClockFrequency": 98304000,
+
+        "inputDataType": {
+            "wordLength": 24,
+            "fractionLength": 23,
+            "signed": true
+        },
+        "outputDataType": {
+            "wordLength": 24,
+            "fractionLength": 23,
+            "signed": true
+        },
+        "numberOfChannels": 2
+    },
+
+    "devices": [
+        {
+            "name": "",
+            "registers": []
```

Comment:
Do we want to create a blank register object inside the array so the users have an example of what keys to fill in for a register?

Reply:
I would not do that as this is meant to be a functioning `model.json` and all registers are expected to actually be valid (this would likely break both device driver generation and avalonWrapper generation at a minimum). We could add a `addRegToModelJson` function that would update the existing `model.json` with a new register to address this, without needing to handle blank registers.

Reply:
If we are expecting them to add registers to `model.json` by hand, that's fine so long as it's easy for them to learn what the register json objects need to look like. 

This is fine as is right now. I think we have considerations to address in terms of how `model.json` will or won't interact with the Simulink model and vice versa, but we can address that after we get all of the downstream code generation utilities working with the new `model.json`.